node/pkg/db/governor_test.go

```diff
@@ -537,24 +608,39 @@ func TestLoadingOldPendingTransfers(t *testing.T) {
 	xfers2, pendings2, err := db.GetChainGovernorDataForTime(logger, now)
 
 	require.Nil(t, err)
-	require.Equal(t, 2, len(xfers2))
+	require.Equal(t, 4, len(xfers2))
 	require.Equal(t, 2, len(pendings2))
 
-	assert.Equal(t, xfer1, xfers2[0])
-	assert.Equal(t, xfer2, xfers2[1])
+	sort.SliceStable(xfers2, func(i, j int) bool {
+		return xfers2[i].Timestamp.Before(xfers2[j].Timestamp)
+	})
+
+	assert.Equal(t, oldXfer1, xfers2[0])
+	assert.Equal(t, newXfer1, xfers2[1])
+	assert.Equal(t, oldXfer2, xfers2[2])
+	assert.Equal(t, newXfer2, xfers2[3])
+
 	assert.Equal(t, pending1.Msg, pendings2[0].Msg)
 	assert.Equal(t, pending2.Msg, pendings2[1].Msg)
 }
 
-func marshalOldTransfer(xfer *Transfer) []byte {
+func marshalOldTransfer(t *Transfer) []byte {
```

Comment:
nit: idiomatically, the `t` variable in a go test function is the `*testing.T` reference. It is even referenced in the [official test docs](https://pkg.go.dev/testing) and all over the codebase.

Shadowing variables like this that are idiomatic and used heavily throughout the existing codebase just cause confusion. Please consider renaming it something != `t`.

Other uses of `t` for `*testing.T` in this codebase:
```bash
$ cd node && git grep -c 't \*testing\.T' | sed 's/:/ /g' | column -t | sort -k2 -rn
pkg/query/request_test.go                         38
pkg/governor/governor_test.go                     26
pkg/query/query_test.go                           22
pkg/query/response_test.go                        16
pkg/db/governor_test.go                           16
pkg/common/scissors_test.go                       13
pkg/watchers/near/watcher_test.go                 10
cmd/ccq/parse_config_test.go                      10
pkg/watchers/evm/blocks_by_timestamp_test.go      9
pkg/db/db_test.go                                 9
pkg/accountant/query_test.go                      9
pkg/watchers/solana/ccq_test.go                   8
pkg/watchers/ibc/watcher_test.go                  8
pkg/governor/mainnet_tokens_test.go               8
pkg/adminrpc/adminserver_test.go                  8
pkg/db/accountant_test.go                         7
pkg/common/chainlock_test.go                      7
pkg/node/reobserve_test.go                        6
pkg/common/guardianset_test.go                    6
pkg/accountant/accountant_test.go                 6
pkg/node/node_test.go                             5
pkg/gwrelayer/gwrelayer_test.go                   5
pkg/governor/mainnet_chains_test.go               5
cmd/spy/spy_test.go                               5
pkg/watchers/sui/watcher_test.go                  4
pkg/watchers/evm/connectors/poller_test.go        4
pkg/watchers/evm/connectors/batch_poller_test.go  4
pkg/publicrpc/publicrpcserver_test.go             4
pkg/p2p/registry_test.go                          4
pkg/common/symmetric_test.go                      4
pkg/watchers/near/nearapi/nearapi_test.go         3
pkg/processor/observation_test.go                 3
pkg/p2p/watermark_test.go                         3
pkg/node/config_file_reader_test.go               3
pkg/accountant/watcher_test.go                    3
pkg/watchers/evm/ccq_test.go                      2
pkg/telemetry/telemetry_test.go                   2
pkg/telemetry/prom_remote_write/prom_test.go      2
pkg/p2p/p2p_test.go                               2
pkg/p2p/cutover_test.go                           2
pkg/devnet/deterministic_p2p_key_test.go          2
pkg/devnet/deterministic_bridge_key_test.go       2
pkg/db/purge_vaas_test.go                         2
pkg/common/nodekey_test.go                        2
pkg/common/mode_test.go                           2
pkg/accountant/submit_obs_test.go                 2
pkg/watchers/evm/utils_test.go                    1
pkg/watchers/algorand/watcher_test.go             1
pkg/supervisor/supervisor_testhelpers.go          1
pkg/processor/backoff_test.go                     1
pkg/p2p/netmetrics_test.go                        1
pkg/node/url_verification_test.go                 1
pkg/governor/governor_monitoring_test.go          1
pkg/common/readiness_test.go                      1
pkg/common/obsvReqSendC_test.go                   1
pkg/accountant/data_for_test.go                   1
hack/query/test/query_test.go                     1
```